apis/sample/v1alpha1/types.go

```diff
@@ -46,13 +46,14 @@ type MyTypeStatus struct {
 
 // +kubebuilder:object:root=true
 
-// A MyType is an example API type
+// A MyType is an example API type.
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:name="STATUS",type="string",JSONPath=".status.bindingPhase"
 // +kubebuilder:printcolumn:name="STATE",type="string",JSONPath=".status.atProvider.state"
 // +kubebuilder:printcolumn:name="CLASS",type="string",JSONPath=".spec.classRef.name"
 // +kubebuilder:printcolumn:name="AGE",type="date",JSONPath=".metadata.creationTimestamp"
-// +kubebuilder:resource:scope=Cluster
+// Please replace `PROVIDER-NAME` with your actual provider name, like `aws`, `azure`, `gcp`, `alibaba`
+// +kubebuilder:resource:scope=Cluster,categories={crossplane,provider,PROVIDER-NAME}
```

Comment:
I think for managed resources we usually include `managed` rather than `provider`.

```suggestion
// +kubebuilder:resource:scope=Cluster,categories={crossplane,managed,PROVIDER-NAME}
```